src/runtime_src/core/common/sensor.cpp

```diff
@@ -214,7 +215,8 @@ read_data_driven_electrical(const std::vector<xq::sdm_sensor_info::data_type>& c
   // iterate over power data, store to ptree by converting to watts.
   for (const auto& tmp : power) {
     if (boost::iequals(tmp.label, "Total Power")) {
-      bd_power = xrt_core::utils::format_base10_shiftdown(tmp.input, tmp.unitm, 3);
+      if (tmp.input != UINT32_MAX)
```

Comment:
I am not sure if this is important, but this code will break if someone changes the type of `xq::sdm_sensor_info::data_type::input`.  
```suggestion
      if (tmp.input != std::numeric_limits<decltype(xq::sdm_sensor_info::data_type::input)>::max())`
```

Reply:
Thank you! I edited the check.